orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/GetCommitsTask.groovy

```diff
@@ -123,7 +123,8 @@ class GetCommitsTask implements DiffTask {
 
       return new TaskResult(ExecutionStatus.SUCCEEDED, outputs)
     } catch (RetrofitError e) {
-      if (e.response?.status == 404) { // just give up on 404
+      if (e.response?.status == 404 || e.kind == RetrofitError.Kind.UNEXPECTED) {
+        // just give up on 404 or internal errors
         log.error("got a 404 from igor for : [repoType: ${repoInfo?.repoType} projectKey:${repoInfo?.projectKey} repositorySlug:${repoInfo?.repositorySlug} sourceCommit:${sourceInfo} targetCommit: ${targetInfo}]")
```

Comment:
error message might be confusing if it's one of the `UNEXPECTED` and not a `404`